test/unit/mock/cmd/bash-c-type-nginx

```diff
@@ -0,0 +1 @@
+nginx is /usr/sbin/nginx
```

Comment:
@adamleff I have resolved the merge conflicts. I am having issue with the mock. Perhaps I can implement it in the future resources if that's ok with you. Thank you!